components/common-data-items.js

```diff
@@ -364,9 +388,37 @@ export function BiosampleDataItems({
           </DataItemValue>
         </>
       )}
+      {pooledIn?.length > 0 && (
+        <>
+          <DataItemLabel>Pooled In Sample(s)</DataItemLabel>
+          <DataItemValue>
+            <SeparatedList>
+              {pooledIn.map((biosample) => (
+                <Link href={biosample["@id"]} key={biosample["@id"]}>
+                  {biosample.accession}
+                </Link>
+              ))}
+            </SeparatedList>
+          </DataItemValue>
+        </>
+      )}
+      {parts?.length > 0 && (
+        <>
+          <DataItemLabel>Sample(s) Parts</DataItemLabel>
+          <DataItemValue>
+            <SeparatedList>
+              {parts.map((biosample) => (
+                <Link href={biosample["@id"]} key={biosample["@id"]}>
+                  {biosample.accession}
+                </Link>
+              ))}
+            </SeparatedList>
+          </DataItemValue>
+        </>
+      )}
       {partOf && (
         <>
-          <DataItemLabel>Part of Biosample</DataItemLabel>
+          <DataItemLabel>Part of Sample(s)</DataItemLabel>
```

Comment:
Similar to above, I’m suprised this can be plural. I might just not understand.

Reply:
confirmed this one is single. fixed.